docs/src/main/sphinx/routines/examples.md

```diff
@@ -0,0 +1,371 @@
+# Example SQL routines
+
+
+After learning about [SQL routines from the
+introduction](/routines/introduction), the following sections show numerous
+examples of valid SQL routines. The routines are suitable as [inline
+routines](routine-inline) or [catalog routines](routine-catalog), after
+adjusting the name and adjusting the example invocations.
```

Comment:
I like this style "as inline routines or as catalog routines". We should use this in the introduction

Reply:
Done